scripts/upgrade-testing.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+CEPHCSI_VERSION=$1
+shift
+test_type=$1
+
+# checkout to respective release branch.
+git checkout "${CEPHCSI_VERSION}" --force
+
+upgrade_cephfs() {
+    # Upgrade CephFS Provisioner resources
+    # Update the CephFS Provisioner RBAC
+    kubectl apply -f deploy/cephfs/kubernetes/csi-provisioner-rbac.yaml
```

Comment:
you need to deploy ceph-csi in a namespace not the default one to upgrade to work properly.